src/utils/DockerMachineUtil.js

```diff
@@ -24,11 +24,11 @@ var DockerMachine = {
   version: function () {
     return util.exec([this.command(), '-v']).then(stdout => {
       try {
-        let tokens = stdout.split(' ');
-        if (tokens.length < 3) {
-          return Promise.resolve(null);
+        var match = stdout.match(/(\d+\.\d+\.\d+).*/);
```

Comment:
Only change I would make is rename the var to something that doesn't match the method name:
`var matched`
vs 
`var match`